README.md
freshfree
=========

Fresh Free Theme


changelog.txt
Fresh Lite Changelog

****************************

Version 1.0.46 (2-12-15)
- Fix few minor issues related to customization

Version 1.0.45 (16-07-15)
- Change in theme customizer as per wordpress.org guidelines

Version 1.0.44 (1-23-15)
- Fixed conflict between slider free and pro plugin.
- Added new plugin suggestion.

Version 1.0.43 (11-25-14)
- Removed theme activation notice.
- Changed default links for slider.

Version 1.0.42 (11-16-14)
- Removed cyberchimps_mlu_init() as it has register_post_type() which is plugin-territory functionality.
- Added sanitization to customizer options.
- Removed deprecated screen_icon().

Version 1.0.41 (9-24-14)
- Placeholder fix for IE8/9.

Version 1.0.40 (9-08-14)
- Added iFeature slider to plugin suggestion list.

Version 1.0.39 (8-22-14)
- Updated Translations.

Version 1.0.38 (8-08-14)
- Fixed conflict of footer widgets with other widgets.
- Responsive fix for uploaded video.

Version 1.0.37 (7-25-14)
- Layout fixed for multiple rows of footer widget.

Version 1.0.36 (7-18-14)
- Fixed broken link to CyberChimps logo.

Version 1.0.35 (7-11-14)
- Replaced logo with text in footer credit link.

Version 1.0.34 (7-04-14)
- Added option for Instagram Icon.

Version 1.0.33 (6-27-14)
- Core Translation Updated.

Version 1.0.32 (6-20-14)
- Code reformatting: Changed spaces to tabs.

Version 1.0.31 (6-13-14)
- Code reformatted: Changed spaces to tabs.

Version 1.0.30 (6-06-14)
- Code reformatting: Changed spaces to tabs.

Version 1.0.29 (5-30-14)
- Fixed casing of CyberChimpsParallax class.

Version 1.0.28 (5-23-14)
- Updated ugrade text.
- Removed left sidebar from widget section.
- Removed "Custom Callout" from sub settings.

Version 1.0.27 (5-16-14)
- Correct spelling of CyberChimps/CSS.
- Change text for Google font field.
- Added cyberchimps_ prefix to couple of functions.
- Changed CyberChimps logo on theme options page.
- Changed "alt" text of logo to the site name instead of "logo".
- Regenerated ".mo" files as they were 0KB each before, hence breaking the translation.

Version 1.0.26 (5-9-14)
- Fixed W3C validator error issue with IE compatibility meta tags.
- Fixed missing image error: removed unused function.
- Added FooBox as recommended plugin.

Version 1.0.25 (5-2-14)
- Fixed byline toggle issue. 

Version 1.0.24 (4-24-14)
- Added recommendation for clef plugin.
- Updated translations.
- Youtube video fixed.

Version 1.0.23 (4-16-14)
- Changed social font icons to font awesome.
- Removed unwanted function_exists() conditional wrapper.

Version 1.0.22 (4-9-14)
-Added upgrade button to the free theme customizer.
-Fixed Vimeo video responsiveness.

Version 1.0.21 (4-3-14)
- Added new background option.
- Removed wp_register_style(or script) for local css/js files.

Version 1.0.20 (3-27-14)
- Updated Elements Translation.

Version 1.0.19 (3-21-14)
- Updated support forum URL.
- Fixed IE8 compatibility issue.

Version 1.0.18 (3-12-14)
- Fixed duplicate plugin install notification in option page.
- Modified "More Themes" page to show up all CyberChimps free theme with option to install/activate right there.

Version 1.0.17 (3-5-14)
- Fixed conflict between blog post excerpt option and bbpress user profile page.
- Set up element options defaults for better wp.org preview.
- Added filter for background default args.
- HTML Error fix: Added space before menu>li>attribute.

Version 1.0.16 (2-27-14)
- Updated elements images along with boxes images.
- Fixed Read more text option.
- Responsive toggle fix: tweaked full-width-container width.
- Background option fix.
- Removed post format icon from page.
- Updated screenshot.

Version 1.0.15 (2-18-14)
- Slider is made to be start sliding on page load.

Version 1.0.14 (2-11-14)
- Updated Translation.

Version 1.0.13 (2-5-14)
- Fixed multiple thumbnails issue on image attachment page.
- Fixed image load issue for boxes element.
- Upsell page tweaked(Font & Text).

Version 1.0.12 (1-27-14)
- Boxes: Added latest default images and retina versions.
- Added Danish translation to core.
- Added hook to set blog layout option default.

Version 1.0.11 (1-21-14)
- Email icon made not to open a new tab.
- Removed Twitter bar from default option.

Version 1.0.10 (1-14-14)
- Changed blog layout option text.
- Fixed upload button not working issue with child themes.

Version 1.0.9 (1-6-14)
- Modified Helvetica font-family.
- Removed translation of a variable.

Version 1.0.8 (12-23-13)
- Fixed drag and drop font icons on page options.
- Added metabox field type unfiltered_textarea.
- Changed underscrore to hyphen in class for portfolio lite element.

Version 1.0.7 (12-16-13)
- Email and google map icon issue fixed.
- Social icon issue in IE fixed.
- Added larger screenshot for WP3.8.
- Update tags for WP3.8.
- Updated license.

Version 1.0.6 (12-12-13)
- Fixed social icon links.
- Fixed metabox toggle option.

Version 1.0.5 (12-11-13)
- Added Font-Based Social Icons.
- Integrated Font Awesome (Bootstrap font).
- Added retina ready images and style for it.
- Changed the way of metabox css/js getting enqueued to support child theme.

Version 1.0.4 (12-04-13)
- Image size fix: Hooked add_image_size to after_setup_theme.
- Deleted unused images.
- Added TGM class to recommend responsive Add Ons.
- Removed Google Analytics option (requested by WordPress).
- Removed theme redirection (requested by WordPress).
- Header restyled so that it can accept longer blog titles

Version 1.0.3 (11-12-13)
- Remove skins in customizer
- Post slides removed
- Translation domain fix
- Sanitization added to few metabox option types
- Js minification in core

Version 1.0.2 (10-04-13)
- License Fixes
- Styling break-word added to stop long words overflowing their containers
- Updated screenshot

Version 1.0.1 (10-30-13)
- License fixes

Version 1.0.0 (10-18-13)
- initial release

image.php
<?php
/**
 * Image Template
 *
 * Please do not edit this file. This file is part of the CyberChimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */

get_header(); ?>

	<div id="image_page" class="container-full-width">

		<div class="container">

			<div class="container-fluid">

				<?php do_action( 'cyberchimps_before_container' ); ?>

				<div id="container" <?php cyberchimps_filter_container_class(); ?>>

					<?php do_action( 'cyberchimps_before_content_container' ); ?>

					<div id="content" <?php cyberchimps_filter_content_class(); ?>>

						<?php do_action( 'cyberchimps_before_content' ); ?>

						<?php while( have_posts() ) : the_post(); ?>

							<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
								<header class="entry-header">
									<h1 class="entry-title"><?php the_title(); ?></h1>

									<div class="entry-meta">
										<?php
										$metadata = wp_get_attachment_metadata();
										printf( __( 'Published', 'cyberchimps' ) . ' <span class="entry-date"><time class="entry-date" datetime="%1$s">%2$s</time></span> - ' . __( 'size', 'cyberchimps' ) . ': <a href="%3$s" title="Link to full-size image">%4$s &times; %5$s</a> ' . __( 'in', 'cyberchimps' ) . ' <a href="%6$s" title="Return to %7$s" rel="gallery">%7$s</a>',
										        esc_attr( get_the_date( 'c' ) ),
										        esc_html( get_the_date() ),
										        wp_get_attachment_url(),
										        $metadata['width'],
										        $metadata['height'],
										        get_permalink( $post->post_parent ),
										        get_the_title( $post->post_parent )
										);
										?>
										<?php edit_post_link( __( 'Edit', 'cyberchimps' ), '<span class="sep"> | </span> <span class="edit-link">', '</span>' ); ?>
									</div>
									<!-- .entry-meta -->

									<nav id="image-navigation">
										<span class="previous-image"><?php previous_image_link( false, '&larr; ' . __( 'Previous', 'cyberchimps' ) ); ?></span>
										<span class="next-image"><?php next_image_link( false, __( 'Next', 'cyberchimps' ) . ' &rarr;' ); ?></span>
									</nav>
									<!-- #image-navigation -->
								</header>
								<!-- .entry-header -->

								<div class="entry-content">

									<div class="entry-attachment">
										<div class="attachment">

											<a href="<?php wp_get_attachment_link( $post->ID, 'fullsize' ); ?>" title="<?php echo esc_attr( get_the_title() ); ?>" rel="attachment"><?php
												$attachment_size = apply_filters( 'cyberchimps_attachment_size', array( 1200, 1200 ) ); // Filterable image size.
												echo wp_get_attachment_image( $post->ID, $attachment_size );
												?></a>
										</div>
										<!-- .attachment -->

										<?php if( !empty( $post->post_excerpt ) ) : ?>
											<div class="entry-caption">
												<?php the_excerpt(); ?>
											</div>
										<?php endif; ?>
									</div>
									<!-- .entry-attachment -->

									<?php the_content(); ?>
									<?php wp_link_pages( array( 'before' => '<div class="page-links">' . __( 'Pages:', 'cyberchimps' ), 'after' => '</div>' ) ); ?>

								</div>
								<!-- .entry-content -->

								<?php
								// HS Thumbnail next and previous image
								$attachments = array_values(
									get_children(
										array(
											'post_parent'    => $post->post_parent,
											'post_status'    => 'inherit',
											'post_type'      => 'attachment',
											'post_mime_type' => 'image',
											'order'          => 'ASC',
											'orderby'        => 'menu_order ID'
										)
									)
								);

								foreach( $attachments as $k => $attachment ) :
									if( $attachment->ID == $post->ID ) {
										break;
									}

									$previous_image = isset( $attachments[$k - 1] ) ? $attachments[$k - 1]->ID : false;
									$next_image     = isset( $attachments[$k + 1] ) ? $attachments[$k + 1]->ID : false;

									$first_image = isset( $attachments[0] ) ? $attachments[0]->ID : false;
									$last_image  = isset( $attachments[$k + 1] ) ? end( $attachments )->ID : false;

									$previous_url = isset( $attachments[$k - 1] ) ? get_permalink( $attachments[$k - 1]->ID ) : get_permalink( $attachments[0]->ID );
									$next_url     = isset( $attachments[$k + 1] ) ? get_permalink( $attachments[$k + 1]->ID ) : get_permalink( $attachments[0]->ID );

									$first_url = isset( $attachments[0] ) ? get_permalink( $attachments[0] ) : false;
									$last_url  = isset( $attachments[$k + 1] ) ? get_permalink( end( $attachments )->ID ) : get_permalink( $attachments[0]->ID );
									?>
									<div class="row-fluid gallery-pagination">
										<div class="span6 previous-image">
											<?php if( $previous_image == false && count( $attachments ) > 1 ): ?>
												<a href="<?php echo $last_url; ?>"><?php echo wp_get_attachment_image( $last_image, 'thumbnail' ); ?></a>
											<?php elseif( $previous_image != $post->ID ): ?>
												<a href="<?php echo $previous_url; ?>"><?php echo wp_get_attachment_image( $previous_image, 'thumbnail' ); ?></a>
											<?php endif; ?>
										</div>
										<!-- span6 -->

										<div class="span6 next-image">
											<?php if( $next_image == false && count( $attachments > 1 ) ): ?>
												<a href="<?php echo $first_url; ?>"><?php echo wp_get_attachment_image( $first_image, 'thumbnail' ); ?></a>
											<?php elseif( $next_image != $post->ID ): ?>
												<a href="<?php echo $next_url; ?>"><?php echo wp_get_attachment_image( $next_image, 'thumbnail' ); ?></a>
											<?php endif; ?>
										</div>
										<!-- span6 -->
									</div><!-- row fluid -->
								<?php endforeach; ?>
								<?php // HS END Thumbnail next and previous image ?>


								<footer class="entry-meta">
									<?php if( comments_open() && pings_open() ) : // Comments and trackbacks open ?>
										<?php printf( '<a class="comment-link" href="#respond" title="Post a comment">' . __( 'Post a comment', 'cyberchimps' ) . '</a> ' . __( 'or leave a trackback', 'cyberchimps' ) . ': <a class="trackback-link" href="%s" title="Trackback URL for your post" rel="trackback">' . __( 'Trackback URL', 'cyberchimps' ) . '</a>.', get_trackback_url() ); ?>
									<?php elseif( !comments_open() && pings_open() ) : // Only trackbacks open ?>
										<?php printf( __( 'Comments are closed, but you can leave a trackback: ', 'cyberchimps' ) . '<a class="trackback-link" href="%s" title="Trackback URL for your post" rel="trackback">' . __( 'Trackback URL', 'cyberchimps' ) . '</a>.', get_trackback_url() ); ?>
									<?php
									elseif( comments_open() && !pings_open() ) : // Only comments open
										?>
										<?php printf( __( 'Trackbacks are closed, but you can %s', 'cyberchimps' ),
										              '<a class="comment-link" href="#respond" title="Post a comment">' . __( 'post a comment', 'cyberchimps' ) . '</a>.' ); ?>
									<?php
									elseif( !comments_open() && !pings_open() ) : // Comments and trackbacks closed
										?>
										<?php _e( 'Both comments and trackbacks are currently closed.', 'cyberchimps' ); ?>
									<?php endif; ?>
									<?php edit_post_link( __( 'Edit', 'cyberchimps' ), ' <span class="edit-link">', '</span>' ); ?>
								</footer>
								<!-- .entry-meta -->
							</article><!-- #post-<?php the_ID(); ?> -->

							<?php comments_template(); ?>

						<?php endwhile; // end of the loop. ?>

						<?php do_action( 'cyberchimps_after_content' ); ?>

					</div>
					<!-- #content -->

					<?php do_action( 'cyberchimps_after_content_container' ); ?>

				</div>
				<!-- #container .row-fluid-->

				<?php do_action( 'cyberchimps_after_container' ); ?>

			</div>
			<!--container fluid -->

		</div>
		<!-- container -->

	</div><!-- container full width -->

<?php get_footer(); ?>

footer.php
<?php
/**
 * Footer Template
 *
 * Please do not edit this file. This file is part of the Cyber Chimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */

if( cyberchimps_get_option( 'footer_show_toggle' ) == '1' ) {

	do_action( 'cyberchimps_before_footer_widgets' ); ?>

	<div class="container-full-width" id="footer_section">
		<div class="container">
			<div class="container-fluid">
				<div id="footer-widgets" class="row-fluid">
					<div id="footer-widget-container" class="span12">
						<div class="row-fluid">

							<?php if( !function_exists( 'dynamic_sidebar' ) || !dynamic_sidebar( 'cyberchimps-footer-widgets' ) ) : ?>
								<aside class="widget-container span3">
									<h3 class="widget-title"><?php _e( 'Pages', 'fresh-lite' ); ?></h3>
									<ul>
										<?php wp_list_pages( 'title_li=' ); ?>
									</ul>
								</aside>

								<aside class="widget-container span3">
									<h3 class="widget-title"><?php _e( 'Archives', 'fresh-lite' ); ?></h3>
									<ul>
										<?php wp_get_archives( 'type=monthly' ); ?>
									</ul>
								</aside>

								<aside class="widget-container span3">
									<h3 class="widget-title"><?php _e( 'Categories', 'fresh-lite' ); ?></h3>
									<ul>
										<?php wp_list_categories( 'show_count=1&title_li=' ); ?>
									</ul>
								</aside>

								<aside class="widget-container span3">
									<h3 class="widget-title"><?php _e( 'WordPress', 'fresh-lite' ); ?></h3>
									<ul>
										<?php wp_register(); ?>
										<li><?php wp_loginout(); ?></li>
										<li><a href="<?php echo esc_url( 'http://wordpress.org/' ); ?>" target="_blank"
										       title="<?php esc_attr_e( 'Powered by WordPress, state-of-the-art semantic personal publishing platform.', 'fresh-lite' ); ?>"> <?php _e( 'WordPress', 'fresh-lite' ); ?></a>
										</li>
										<?php wp_meta(); ?>
									</ul>
								</aside>
							<?php endif; ?>
						</div>
						<!-- .row-fluid -->
					</div>
					<!-- #footer-widget-container -->
				</div>
				<!-- #footer-widgets .row-fluid  -->
			</div>
			<!-- .container-fluid-->
		</div>
		<!-- .container -->
	</div>    <!-- #footer_section -->

	<?php do_action( 'cyberchimps_after_footer_widgets' ); ?>

<?php } ?>

<?php do_action( 'cyberchimps_before_footer_container' ); ?>
<?php do_action( 'cyberchimps_footer' ); ?>
<?php do_action( 'cyberchimps_after_footer_container' ); ?>

</div><!-- #wrapper .container-fluid -->

<?php do_action( 'cyberchimps_after_wrapper' ); ?>

</div><!-- container -->

<?php wp_footer(); ?>

</body>
</html>

header.php
<?php
/**
 * Header Template
 *
 * Please do not edit this file. This file is part of the Cyber Chimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */
?>
	<!DOCTYPE html>
	<meta http-equiv="X-UA-Compatible" content="IE=10;IE=9;IE=8;IE=EDGE" />
	<!--[if lt IE 7]>
	<html class="ie ie6 lte9 lte8 lte7" <?php language_attributes(); ?>>
	<![endif]-->
	<!--[if IE 7]>
	<html class="ie ie7 lte9 lte8 lte7" <?php language_attributes(); ?>>
	<![endif]-->
	<!--[if IE 8]>
	<html class="ie ie8 lte9 lte8" <?php language_attributes(); ?>>
	<![endif]-->
	<!--[if IE 9]>
	<html class="ie ie9" <?php language_attributes(); ?>>
	<![endif]-->
	<!--[if gt IE 9]>
	<html <?php language_attributes(); ?>> <![endif]-->
	<!--[if !IE]><!-->
<html <?php language_attributes(); ?>>
	<!--<![endif]-->
	<head>
		<meta charset="<?php bloginfo( 'charset' ); ?>"/>
		<meta name="viewport" content="initial-scale=1.0,maximum-scale=3.0,width=device-width"/>

		<title><?php wp_title( '' ); ?></title>

		<link rel="profile" href="http://gmpg.org/xfn/11"/>
		<link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>"/>

		<!-- IE6-8 support of HTML5 elements -->
		<!--[if lt IE 9]>
		<script src="<?php echo get_template_directory_uri(); ?>/inc/js/html5.js" type="text/javascript"></script>
		<![endif]-->

		<?php wp_head(); ?>
	</head>
<body <?php body_class(); ?>>

<!-- ---------------- Header --------------------- -->
<div class="container-full-width" id="header_section">
	<div class="container">
		<?php do_action( 'cyberchimps_before_wrapper' ); ?>
		<div class="container-fluid">
			<?php do_action( 'cyberchimps_header' ); ?>
		</div>
		<!-- .container-fluid-->
	</div>
	<!-- .container -->
</div>
<!-- #header_section -->

<?php do_action( 'cyberchimps_before_navigation' ); ?>

<!-- ---------------- Menu --------------------- -->
<div class="container-full-width" id="navigation_menu">
	<div class="container">
		<div class="container-fluid">
			<nav id="navigation" role="navigation">
				<div class="main-navigation navbar">
					<div class="navbar-inner">
						<div class="container">

							<?php /* hide collapsing menu if not responsive */
							if (cyberchimps_get_option( 'responsive_design', 'checked' )): ?>
							<div class="nav-collapse collapse">
								<?php endif; ?>

								<?php wp_nav_menu( array( 'theme_location' => 'primary', 'menu_class' => 'nav', 'walker' => new cyberchimps_walker(), 'fallback_cb' => 'cyberchimps_fallback_menu' ) ); ?>

								<?php if( cyberchimps_get_option( 'searchbar', 1 ) == "1" ) : ?>
									<?php get_search_form(); ?>
								<?php endif; ?>

								<?php /* hide collapsing menu if not responsive */
								if (cyberchimps_get_option( 'responsive_design', 'checked' )): ?>
							</div>
						<!-- collapse -->

						<!-- .btn-navbar is used as the toggle for collapsed navbar content -->
							<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
								<span class="icon-bar"></span>
								<span class="icon-bar"></span>
								<span class="icon-bar"></span>
							</a>
						<?php endif; ?>
						</div>
						<!-- container -->
					</div>
					<!-- .navbar-inner .row-fluid -->
				</div>
				<!-- main-navigation navbar -->
			</nav>
			<!-- #navigation -->
		</div>
		<!-- .container-fluid-->
	</div>
	<!-- .container -->
</div>
<!-- #navigation_menu -->

<?php do_action( 'cyberchimps_after_navigation' ); ?>

comments.php
<?php
/**
 * Comments Template
 *
 * Please do not edit this file. This file is part of the Cyber Chimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category CyberChimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */
?>

<?php
/*
 * If the current post is protected by a password and
 * the visitor has not yet entered the password we will
 * return early without loading the comments.
 */
if( post_password_required() ) {
	return;
}
?>

<div id="comments" class="comments-area">

	<?php // You can start editing here -- including this comment! ?>

	<?php if( have_comments() ) : ?>
		<h2 class="comments-title">
			<?php
			printf( _n( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'cyberchimps' ),
			        number_format_i18n( get_comments_number() ), '<span>' . get_the_title() . '</span>' );
			?>
		</h2>

		<?php if( get_comment_pages_count() > 1 && get_option( 'page_comments' ) ) : // are there comments to navigate through ?>
			<nav role="navigation" id="comment-nav-above" class="site-navigation comment-navigation">
				<h4 class="assistive-text"><?php _e( 'Comment navigation', 'cyberchimps' ); ?></h4>

				<div class="nav-previous"><?php previous_comments_link( '&larr; ' . __( 'Older Comments', 'cyberchimps' ) ); ?></div>
				<div class="nav-next"><?php next_comments_link( __( 'Newer Comments', 'cyberchimps' ) . ' &rarr;' ); ?></div>
			</nav>
		<?php endif; // check for comment navigation ?>

		<ol class="commentlist">
			<?php
			/* Loop through and list the comments. Tell wp_list_comments()
			 * to use cyberchimps_comment() to format the comments.
			 * If you want to overload this in a child theme then you can
			 * define cyberchimps_comment() and that will be used instead.
			 * See cyberchimps_comment() in functions.php for more.
			 */
			wp_list_comments( array( 'callback' => 'cyberchimps_comment' ) );
			?>
		</ol>

		<?php if( get_comment_pages_count() > 1 && get_option( 'page_comments' ) ) : // are there comments to navigate through ?>
			<nav role="navigation" id="comment-nav-below" class="site-navigation comment-navigation">
				<h4 class="assistive-text"><?php _e( 'Comment navigation', 'cyberchimps' ); ?></h4>

				<div class="nav-previous"><?php previous_comments_link( '&larr; ' . __( 'Older Comments', 'cyberchimps' ) ); ?></div>
				<div class="nav-next"><?php next_comments_link( __( 'Newer Comments', 'cyberchimps' ) . ' &rarr;' ); ?></div>
			</nav>
		<?php endif; // check for comment navigation ?>

	<?php endif; // have_comments() ?>

	<?php
	// If comments are closed and there are comments, let's leave a little note, shall we?
	if( !comments_open() && '0' != get_comments_number() && post_type_supports( get_post_type(), 'comments' ) ) :
		?>
		<p class="nocomments"><?php _e( 'Comments are closed.', 'cyberchimps' ); ?></p>
	<?php endif; ?>

	<?php comment_form(); ?>

</div><!-- #comments .comments-area -->

functions.php
<?php
/**
 * Title: Function
 *
 * Description: Defines theme specific functions including actions and filters.
 *
 * Please do not edit this file. This file is part of the Cyber Chimps Framework and all modifications
 * should be made in a child theme.
 *
 * @category Cyber Chimps Framework
 * @package  Framework
 * @since    1.0
 * @author   CyberChimps
 * @license  http://www.opensource.org/licenses/gpl-license.php GPL v3.0 (or later)
 * @link     http://www.cyberchimps.com/
 */

// Load text domain.
function cyberchimps_text_domain() {
	load_theme_textdomain( 'fresh-lite', get_template_directory() . '/inc/languages' );
}
add_action( 'after_setup_theme', 'cyberchimps_text_domain' );

// Load Core
require_once( get_template_directory() . '/cyberchimps/init.php' );

// Set the content width based on the theme's design and stylesheet.
if( !isset( $content_width ) ) {
	$content_width = 640;
} /* pixels */

// Define site info
function cyberchimps_add_site_info() {
	?>
	<p>&copy; Company Name</p>
<?php
}

add_action( 'cyberchimps_site_info', 'cyberchimps_add_site_info' );

if( !function_exists( 'cyberchimps_comment' ) ) :

// Template for comments and pingbacks.
// Used as a callback by wp_list_comments() for displaying the comments.
	function cyberchimps_comment( $comment, $args, $depth ) {
		$GLOBALS['comment'] = $comment;
		switch( $comment->comment_type ) :
			case 'pingback' :
			case 'trackback' :
				?>
				<li class="post pingback">
				<p><?php _e( 'Pingback:', 'fresh-lite' ); ?> <?php comment_author_link(); ?><?php edit_comment_link( __( '(Edit)', 'fresh-lite' ), ' ' ); ?></p>
				<?php
				break;
			default :
				?>
					<li <?php comment_class(); ?> id="li-comment-<?php comment_ID(); ?>">
					<article id="comment-<?php comment_ID(); ?>" class="comment hreview">
						<footer>
							<div class="comment-author reviewer vcard">
								<?php echo get_avatar( $comment, 40 ); ?>
								<?php printf( '%s <span class="says">' . __( 'says:', 'fresh-lite' ) . '</span>', sprintf( '<cite class="fn">%s</cite>', get_comment_author_link() ) ); ?>
							</div>
							<!-- .comment-author .vcard -->
							<?php if( $comment->comment_approved == '0' ) : ?>
								<em><?php _e( 'Your comment is awaiting moderation.', 'fresh-lite' ); ?></em>
								<br/>
							<?php endif; ?>

							<div class="comment-meta commentmetadata">
								<a href="<?php echo esc_url( get_comment_link( $comment->comment_ID ) ); ?>" class="dtreviewed">
									<time pubdate datetime="<?php comment_time( 'c' ); ?>">
										<?php
										/* translators: 1: date, 2: time */
										printf( __( '%1$s at %2$s', 'fresh-lite' ), get_comment_date(), get_comment_time() ); ?>
									</time>
								</a>
								<?php edit_comment_link( __( '(Edit)', 'fresh-lite' ), ' ' );
								?>
							</div>
							<!-- .comment-meta .commentmetadata -->
						</footer>

						<div class="comment-content"><?php comment_text(); ?></div>

						<div class="reply">
							<?php comment_reply_link( array_merge( $args, array( 'depth' => $depth, 'max_depth' => $args['max_depth'] ) ) ); ?>
						</div>
						<!-- .reply -->
					</article><!-- #comment-## -->

				<?php
				break;
		endswitch;
	}
endif; // ends check for cyberchimps_comment()

// set up next and previous post links for lite themes only
function cyberchimps_next_previous_posts() {
	if( get_next_posts_link() || get_previous_posts_link() ): ?>
		<div class="more-content">
			<div class="row-fluid">
				<div class="span6 previous-post">
					<?php previous_posts_link(); ?>
				</div>
				<div class="span6 next-post">
					<?php next_posts_link(); ?>
				</div>
			</div>
		</div>
	<?php
	endif;
}

add_action( 'cyberchimps_after_content', 'cyberchimps_next_previous_posts' );

// core options customization Names and URL's
//Pro or Free
function cyberchimps_theme_check() {
	$level = 'free';

	return $level;
}

//Theme Name
function cyberchimps_options_theme_name() {
	$text = 'Fresh Free';

	return $text;
}

//Theme Pro Name
function cyberchimps_upgrade_bar_pro_title() {
	$text = 'Fresh Pro';

	return $text;
}

//Upgrade link
function cyberchimps_upgrade_bar_pro_link() {
	$url = 'http://cyberchimps.com/store/fresh-pro/';

	return $url;
}

//Doc's URL
function cyberchimps_options_documentation_url() {
	$url = 'http://cyberchimps.com/guides/c/';

	return $url;
}

// Support Forum URL
function cyberchimps_options_support_forum() {
	$url = 'http://cyberchimps.com/forum/free/freshfree/';

	return $url;
}

add_filter( 'cyberchimps_current_theme_name', 'cyberchimps_options_theme_name', 1 );
add_filter( 'cyberchimps_upgrade_pro_title', 'cyberchimps_upgrade_bar_pro_title' );
add_filter( 'cyberchimps_upgrade_link', 'cyberchimps_upgrade_bar_pro_link' );
add_filter( 'cyberchimps_documentation', 'cyberchimps_options_documentation_url' );
add_filter( 'cyberchimps_support_forum', 'cyberchimps_options_support_forum' );

// Help Section
function cyberchimps_options_help_header() {
	$text = 'Fresh Free';

	return $text;
}

function cyberchimps_options_help_sub_header() {
	$text = __( 'Fresh Free Professional Responsive WordPress Theme', 'fresh-lite' );

	return $text;
}

add_filter( 'cyberchimps_help_heading', 'cyberchimps_options_help_header' );
add_filter( 'cyberchimps_help_sub_heading', 'cyberchimps_options_help_sub_header' );

// Branding images and defaults

// Banner default
function cyberchimps_banner_default() {
	$url = '/images/branding/banner.jpg';

	return $url;
}

add_filter( 'cyberchimps_banner_img', 'cyberchimps_banner_default' );

//theme specific skin options in array. Must always include option default
function cyberchimps_skin_color_options( $options ) {
	// Get path of image
	$imagepath = get_template_directory_uri() . '/inc/css/skins/images/';

	$options = array(
		'default' => $imagepath . 'default.png'
	);

	return $options;
}

add_filter( 'cyberchimps_skin_color', 'cyberchimps_skin_color_options' );

// theme specific background images
function cyberchimps_background_image( $options ) {
	$imagepath = get_template_directory_uri() . '/cyberchimps/lib/images/';
	$options   = array(
		'none'  => $imagepath . 'backgrounds/thumbs/none.png',
		'noise' => $imagepath . 'backgrounds/thumbs/noise.png',
		'blue'  => $imagepath . 'backgrounds/thumbs/blue.png',
		'dark'  => $imagepath . 'backgrounds/thumbs/dark.png',
		'space' => $imagepath . 'backgrounds/thumbs/space.png'
	);

	return $options;
}

add_filter( 'cyberchimps_background_image', 'cyberchimps_background_image' );

// theme specific typography options
function cyberchimps_typography_sizes( $sizes ) {
	$sizes = array( '8', '9', '10', '12', '14', '16', '20' );

	return $sizes;
}

function cyberchimps_typography_faces( $orig ) {

	$new = array(
		'"Noto Sans", Arial, sans-serif'          => 'Noto Sans',
		'"Imprima", Helvetica, Arial, sans-serif' => 'Imprima'
	);

	$new = array_merge( $new, $orig );

	return $new;
}

function cyberchimps_typography_styles( $styles ) {
	$styles = array( 'normal' => 'Normal', 'bold' => 'Bold' );

	return $styles;
}

function cyberchimps_typography_defaults() {
	$default = array(
		'size'  => '14px',
		'face'  => '"Noto Sans", Arial, sans-serif',
		'style' => 'normal',
		'color' => '#555555'
	);

	return $default;
}

function cyberchimps_typography_heading_defaults() {
	$default = array(
		'size'  => '',
		'face'  => '"Imprima", Helvetica, Arial, sans-serif',
		'style' => '',
		'color' => '',

	);

	return $default;
}

add_filter( 'cyberchimps_typography_sizes', 'cyberchimps_typography_sizes' );
add_filter( 'cyberchimps_typography_styles', 'cyberchimps_typography_styles' );
add_filter( 'cyberchimps_typography_faces', 'cyberchimps_typography_faces' );
add_filter( 'cyberchimps_typography_defaults', 'cyberchimps_typography_defaults' );
add_filter( 'cyberchimps_typography_heading_defaults', 'cyberchimps_typography_heading_defaults' );

// Default for twitter bar handle
function cyberchimps_twitter_handle_filter() {
	return 'WordPress';
}

add_filter( 'cyberchimps_twitter_handle_filter', 'cyberchimps_twitter_handle_filter' );